wsf_scraping/spiders/nice_spider.py

```diff
@@ -0,0 +1,227 @@
+import os
+import json
+import scrapy
+import logging
+from lxml import html
+from scrapy.http import Request
+from wsf_scraping.items import NICEArticle
+from tools.dbTools import check_db, is_scraped
+
+
+class NiceSpider(scrapy.Spider):
+    name = 'nice'
+
+    def start_requests(self):
+        """Set up the initial request to the website to scrape."""
+
+        urls = []
+        articles_count = self.settings['NICE_ARTICLES_COUNT']
+
+        # Initial URL (splited for PEP8 compliance). -1 length displays
+        # the whole list.
+        base_url = 'https://www.nice.org.uk/guidance/published/ajax'
+        param1 = '?iDisplayLength=%s' % articles_count
+        param2 = '&type=apg,csg,cg,mpg,ph,sg,sc'
+        url = ''.join([base_url, param1, param2])
+
+        # NICE guidance listing should be called by an AJAX requests
+        headers = {
+            'X-Requested-With': 'XMLHttpRequest',
+            'referer': 'https://www.nice.org.uk/guidance/published'
```

Comment:
This is duplicated from the Ajax contract. Is there any way to refactor?

Reply:
Not the same purpose. The contract is a test, where this is the real request

Reply:
👍